src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -368,6 +368,8 @@ curl -X POST https://your.endpoint/cancel \
 
 After initiating a payment transaction, the status may be `pending`. This endpoint will be used to poll the status of the transaction attempt. Polling will continue until either the transaction attempt status is `successful` or `failed`, or the Centrapay Payment Request is no longer payable (e.g. it has expired).
 
+We will also use this endpoint to resolve transactions if we are unsure of the status of a transaction attempt. If the transaction does not exist in your system a 2XX response with an empty object as the body should be returned.
```

Comment:
@AndyClifford These changes feel like they've been tacked on as an after thought (multiple uses of `also`). 

Suggestion: Make important details clearer and make this endpoint description more succinct. I'd consider at rewriting this entire endpoint description.

> This endpoint is used to poll the Transaction Attempt status when it is `pending` or unknown due to an unexpected behavior. 
>
> Polling will continue until either the transaction attempt status is `successful` or `failed`, or the Centrapay Payment Request is no longer payable (e.g. it has expired).
>
> You should return an empty 2XX response if the `transactionId` does not exist in your system.